OasisDEX/oasis-borrow
# Close to Coll Swap Fix

# [Close to Coll Swap Fix](https://app.shortcut.com/oazo-apps/story/10589/close-to-coll-swap-fix)
  
## Changes 👷‍♀️
- Integrate library with close to coll fix
  
## How to test 🧪
- Go to `/ethereum/ajna/multiply/USDC-ETH`
- Open a position
- Close position with close to collateral (observe that the balances displayed are as expected)
